docs/readthedocs/models/dhws_model.md

```diff
@@ -0,0 +1,56 @@
+(dhws_model)=
+# Domestic Hot Water Storage
+This page documents the functionality of the domestic hot water storage available in SIMONA.
+
+## Behaviour
+This storage model operates on volumes, although the functions it provides for other models all operate with energy. Internally the storage model converts energy to volume and vice versa with formulas specified below. Furthermore it is assumed that the storage medium is water. Also the model holds a variable for the current storage level. At initialisation the storage will be full. Domestic hot water storage will be refilled when the state of charge will be below 20 % or when the required water demand can not be covered by the stored energy in the storage itself.
+
```

Comment:
This should be described more clearly.

Reply:
Do you have a proposal or could you describe a bit what is unclear to you?

Reply:
There can be situations (high demand of tap water because of many people or DomesticHotWaterStorage with very small volume) where the energy demand for tap water is higher than the acutal energy level stored. In this case the heat source will be actived directly even if the storage SoC is above 20 % and will try to cover of course first the tap water demand and, if there is remaining power, to recharge the storage.